docs/concepts.rst

```diff
@@ -737,28 +737,28 @@ dependency settings.
 
 All operators have a ``trigger_rule`` argument which defines the rule by which
 the generated task get triggered. The default value for ``trigger_rule`` is
-``all_success`` and can be defined as "trigger this task when all directly
+``TriggerRule.ALL_SUCCESS`` and can be defined as "trigger this task when all directly
 upstream tasks have succeeded". All other rules described here are based
 on direct parent tasks and are values that can be passed to any operator
 while creating tasks:
```

Comment:
We probably need to mention in this doc that these aren't strings but constants/enums from module X.